Fixing the default kea config to put all logs under /var/log/kea directory.
--- a/src/bin/keactrl/kea-ctrl-agent.conf.pre
+++ b/src/bin/keactrl/kea-ctrl-agent.conf.pre
@@ -66,7 +66,7 @@
                 // - syslog (logs to syslog)
                 // - syslog:name (logs to syslog using specified name)
                 // Any other value is considered a name of a time
-                "output": "@localstatedir@/log/kea-ctrl-agent.log"
+                "output": "/var/log/kea/ctrl-agent.log"
 
                 // This governs whether the log output is flushed to disk after
                 // every write.
--- a/src/bin/keactrl/kea-dhcp-ddns.conf.pre
+++ b/src/bin/keactrl/kea-dhcp-ddns.conf.pre
@@ -43,7 +43,7 @@
                 // - syslog (logs to syslog)
                 // - syslog:name (logs to syslog using specified name)
                 // Any other value is considered a name of a time
-                "output": "@localstatedir@/log/kea-ddns.log"
+                "output": "/var/log/kea/ddns.log"
 
                 // This governs whether the log output is flushed to disk after
                 // every write.
--- a/src/bin/keactrl/kea-dhcp4.conf.pre
+++ b/src/bin/keactrl/kea-dhcp4.conf.pre
@@ -431,7 +431,7 @@
                 // - syslog (logs to syslog)
                 // - syslog:name (logs to syslog using specified name)
                 // Any other value is considered a name of a time
-                "output": "@localstatedir@/log/kea-dhcp4.log"
+                "output": "/var/log/kea/dhcp4.log"
 
                 // This governs whether the log output is flushed to disk after
                 // every write.
--- a/src/bin/keactrl/kea-dhcp6.conf.pre
+++ b/src/bin/keactrl/kea-dhcp6.conf.pre
@@ -357,7 +357,7 @@
                 // - syslog (logs to syslog)
                 // - syslog:name (logs to syslog using specified name)
                 // Any other value is considered a name of a time
-                "output": "@localstatedir@/log/kea-dhcp6.log"
+                "output": "/var/log/kea/dhcp6.log"
 
                 // This governs whether the log output is flushed to disk after
                 // every write.
